### ISSUE TITLE
question about the Neural network explosion

### ISSUE THREAD

**Issue by zhangtzq** [vishal3477/Reverse_Engineering_GMs#17]
Hi, I have reproduced the code for the image attribution. I get the neural network explosion during training. Surprisingly, I got an accuracy of about 85% before the explosion. I want to know why. I observe that when the explosion happens, the weights of the net obtain nan and the CE loss increases. Can you help me to solve the issue? I used the Adam optimizer.

**Comment by vishal3477** [comment]
Hi,
Can you provide more details about the error? Usually, because of using fft functions of pytorch. I would advise you to restart from the last good checkpoint with a lower learning rate to not allow the model change much.
